features/nanostack/mbed-mesh-api/source/WisunBorderRouter.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2020 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "WisunInterface.h"
+#include "ns_trace.h"
+#include "WisunBorderRouter.h"
+#include "MeshInterfaceNanostack.h"
+
+extern "C" {
+#include "ws_bbr_api.h"
+}
+
+#define TRACE_GROUP "WSBR"
+
+mesh_error_t WisunBorderRouter::start(NetworkInterface *mesh_if, NetworkInterface *backbone_if)
+{
+    InterfaceNanostack *nano_mesh_if = reinterpret_cast<InterfaceNanostack *>(mesh_if);
+    int8_t mesh_if_id = nano_mesh_if->get_interface_id();
```

Comment:
Is there null check needed it could be in if?

Reply:
added null checks